src/workerd/api/js-rpc-target.c++

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2017-2023 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+
+#include <workerd/api/js-rpc-target.h>
+#include <workerd/api/global-scope.h>
+
+namespace workerd::api {
+
+kj::Promise<void> JsRpcTargetImpl::call(CallContext callContext) {
+  auto methodName = kj::heapString(callContext.getParams().getMethodName());
+  // TODO(now): Deserialize the arguments.
+  //    - Current demo doesn't take args, once we pass args on the client side I'll deserialize
+  //      on the server side.
+  auto serializedArgs = callContext.getParams().getSerializedArgs().getV8Serialized().asBytes();
+
+  // Try to execute the requested method.
+  try {
+    auto& ctx = incomingRequest->getContext();
+    co_await ctx.run(
+        [methodName=kj::mv(methodName),
+         serializedArgs = kj::mv(serializedArgs),
+         entrypointName = kj::mv(entrypointName),
+         &ctx,
+         &callContext] (Worker::Lock& lock) mutable {
+
+      auto reader = callContext.initResults(capnp::MessageSize { 4, 1 }).initResult();
+
+      auto handler = lock.getExportedHandler(kj::mv(entrypointName), ctx.getActor());
+      KJ_IF_MAYBE(h, handler) {
+        jsg::Lock& js = lock;
+
+        auto handle = h->self.getHandle(lock);
+        auto methodStr = jsg::v8StrIntern(lock.getIsolate(), methodName);
+        auto methodHandle = jsg::check(handle->Get(lock.getContext(), methodStr));
+        // TODO(now): In this case, we probably want to return an error to the caller?
+        KJ_REQUIRE(methodHandle->IsFunction(), "Unknown function?", methodName);
```

Comment:
These should probably be `JSG_REQUIRE`s -- it's an application error, not something we want to log to Sentry.